{
  "cve": {
    "data_type": "CVE",
    "data_format": "MITRE",
    "data_version": "4.0",
    "CVE_data_meta": {
      "ID": "CVE-2008-0299"
    },
    "affects": {
      "vendor": {
        "vendor_data": [
          {
            "vendor_name": "python_software_foundation",
            "product": {
              "product_data": [
                {
                  "product_name": "paramiko",
                  "version": {
                    "version_data": [
                      {
                        "version_value": "1.7.1"
                      }
                    ]
                  }
                }
              ]
            }
          }
        ]
      }
    },
    "problemtype": {
      "problemtype_data": [
        {
          "description": [
            {
              "lang": "en",
              "value": "NVD-CWE-Other"
            }
          ]
        }
      ]
    },
    "references": {
      "reference_data": [
        {
          "url": "http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=460706"
        },
        {
          "url": "http://people.debian.org/~nion/nmu-diff/paramiko-1.6.4-1_1.6.4-1.1.patch"
        },
        {
          "url": "http://secunia.com/advisories/28488"
        },
        {
          "url": "http://secunia.com/advisories/28510"
        },
        {
          "url": "http://secunia.com/advisories/29168"
        },
        {
          "url": "http://security.gentoo.org/glsa/glsa-200803-07.xml"
        },
        {
          "url": "http://www.lag.net/pipermail/paramiko/2008-January/000599.html"
        },
        {
          "url": "http://www.securityfocus.com/bid/27307"
        },
        {
          "url": "https://bugzilla.redhat.com/show_bug.cgi?id=428727"
        },
        {
          "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/39749"
        },
        {
          "url": "https://www.redhat.com/archives/fedora-package-announce/2008-January/msg00529.html"
        },
        {
          "url": "https://www.redhat.com/archives/fedora-package-announce/2008-January/msg00594.html"
        }
      ]
    },
    "description": {
      "description_data": [
        {
          "lang": "en",
          "value": "common.py in Paramiko 1.7.1 and earlier, when using threads or forked processes, does not properly use RandomPool, which allows one session to obtain sensitive information from another session by predicting the state of the pool."
        }
      ]
    }
  },
  "configurations": {
    "CVE_data_version": "4.0",
    "nodes": [
      {
        "operator": "OR",
        "cpe": [
          {
            "vulnerable": true,
            "cpeMatchString": "cpe:/a:python_software_foundation:paramiko:1.7.1",
            "cpe23Uri": "cpe:2.3:a:python_software_foundation:paramiko:1.7.1:*:*:*:*:*:*:*"
          }
        ]
      }
    ]
  },
  "impact": {
    "baseMetricV2": {
      "cvssV2": {
        "vectorString": "(AV:N/AC:M/Au:N/C:P/I:N/A:N)",
        "accessVector": "NETWORK",
        "accessComplexity": "MEDIUM",
        "authentication": "NONE",
        "confidentialityImpact": "PARTIAL",
        "integrityImpact": "NONE",
        "availabilityImpact": "NONE",
        "baseScore": 4.3
      },
      "severity": "MEDIUM",
      "exploitabilityScore": 8.6,
      "impactScore": 2.9,
      "obtainAllPrivilege": false,
      "obtainUserPrivilege": false,
      "obtainOtherPrivilege": false,
      "userInteractionRequired": false
    }
  },
  "publishedDate": "2008-01-16T23:00Z",
  "lastModifiedDate": "2017-08-08T01:29Z"
}